bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -122,7 +123,7 @@ wf.add_workflow_settings_cli(parser, include_subdax_opts=True)
 ppu.pygrb_add_bestnr_cut_opt(parser)
 args = parser.parse_args()
 
-init_logging(args.verbose, format="%(asctime)s: %(levelname)s: %(message)s")
+pycbc.init_logging(args.verbose, format="%(asctime)s: %(levelname)s: %(message)s")
```

Comment:
Again, is the format what we want here? I notice these are pygrb files so they might be what is needed.

Reply:
Checking `__init__.py` the default for the format option is 'format='%(asctime)s %(levelname)s : %(message)s'):' which when compared to this one is missing a colon after '(asctime)s '

Reply:
Yup, didnt want to break things, just in case